_rules/SC2-4-9-links-with-identical-accessible-names-have-same-purpose.md

```diff
@@ -0,0 +1,234 @@
+---
+name: Links with identical accessible names have same purpose
+test_type: atomic
+
+description: |
+  This rule checks that identical accessible names are only used for links that have the same purpose
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of link, are [exposed to assistive technologies](#exposed-to-assistive-technologies) and that have the same [accessible name](#accessible-name).
+
+### Expectation
+
+When activated, the links in each set of target elements resolve to resources that fulfill the same purpose indicated by the [accessible names](#accessible-name) of the link.
```

Comment:
These are 2 expectations:
1 - resolve to resources that fulfil the same purpose
2 - the purpose is indicated by the accessible name
I don't think the second should be addressed in this rule.

Reply:
@carlosapaduarte, I agree that the second one should not be part of this rule. However, the point here is that "the same purpose" needs to be interpreted in the context of the accessible name.
---
Example:  

Page A: only has a phone number on it
Page B: has a phone number, email, fax, address, etc.

Example 1: Link text is "Call us" --> Rule is Passed, since Page A and Page B fulfil the same purpose
Example 2: Link text is "See all the different ways that you can contact us" --> Rule is Failed, since Page A and Page B doesn't fulfil the same purpose. 
---
Would one of these be better/clearer: 

"fulfil the same purpose in relation to the [accessible names](#accessible-name) of the link."
"fulfil the same purpose, indicated by the [accessible names](#accessible-name) of the link."
"fulfil the same purpose as indicated by the [accessible names](#accessible-name) of the link."